crescent/assetlist.json

```diff
@@ -31,7 +31,7 @@
       "denom_units": [
         {
           "denom": "ubcre",
-          "exponent": 0,
+          "exponent": 6,
```

Comment:
This should remain at 0, as all base denoms do. bcre will have exponent 6, but ubcre should have 0.